docs/Post-Action-Registry.md

```diff
@@ -487,3 +488,33 @@ Adds a new JSON property in an existing JSON file.
   `"continueOnError": true`
 `}]
 ```
+
+# Create or update dotnet.config file
+
+Creates a new `dotnet.config` file with the specified section, key, and value, or updates an existing `dotnet.config`.
+
+- **Action ID** : 597E7933-0D87-452C-B094-8FA0EEF7FD97
+- **Specific Configuration** :
+   - `args`:
+      - `section (string)`: The section name (for example, `dotnet.test.runner`).
+      - `key (string)`: The key to set under the given section (for example, `name`).
+      - `value (string)`: The value to set for the given key (for example, `Microsoft.Testing.Platform`).
+- **Supported in** :
+   - dotnet new3
+   - dotnet new (2.0.0 or higher)
```

Comment:
I'm not sure of this. I just copied it from above post actions

Reply:
I don't think `dotnet new3` exists anymore.